hello-world/tutorial/index.md

```diff
@@ -0,0 +1,494 @@
+		return nil
+	case *channeltypes.Acknowledgement_Result:
+		// Decode the packet acknowledgment
+		var packetAck types.IbcPostPacketAck
+		err := packetAck.Unmarshal(dispatchedAck.Result)
+		if err != nil {
+			// The counter-party module doesn't implement the correct acknowledgment format
+			return errors.New("cannot unmarshal acknowledgment")
+		}
+
+		k.AppendSentPost(
+			ctx,
+			data.Creator,
+			packetAck.PostID,
+			data.Title,
+			packet.DestinationPort+"-"+packet.DestinationChannel,
+		)
+
+		return nil
+	default:
+		// The counter-party module doesn't implement the correct acknowledgment format
+		return errors.New("invalid acknowledgment format")
+	}
+}
+```
+
+### Store Information about the Timed-out Packet
+
+Store posts that have not been received by target chains in `timedoutPost` posts. This logic follows the same format as `sentPost`.
+
+```go
+// OnTimeoutIbcPostPacket responds to the case where a packet has not been transmitted because of a timeout
+func (k Keeper) OnTimeoutIbcPostPacket(ctx sdk.Context, packet channeltypes.Packet, data types.IbcPostPacketData) error {
+
+	k.AppendTimedoutPost(
+		ctx,
+		data.Creator,
+		data.Title,
+		packet.DestinationPort+"-"+packet.DestinationChannel,
+	)
+
+	return nil
+}
+```
+
+This is the basic `blog` module setup.
+The blockchain is now ready!
+
+Spin up the blockchain and send a blog post from one blockchain app to the other.
+
+Various terminal windows are required to complete these next steps.
+
+## Test the IBC modules
+
+To test the IBC module, start two blockchain networks on the same machine. Both blockchains use the same source code. Each blockchain has a unique chain ID.
+
+One blockchain is named `earth` and the other blockchain is named `mars`.
+
+The following files are required in the project directory:
+
+`planet/earth.yml`
+
+```yaml
+accounts:
+  - name: alice
+    coins: ["1000token", "100000000stake"]
+  - name: bob
+    coins: ["500token"]
+validator:
+  name: alice
+  staked: "100000000stake"
+faucet:
+  name: bob
+  coins: ["5token"]
+genesis:
+  chain_id: "earth"
+init:
+  home: "$HOME/.earth"
+```
+
+`planet/mars.yml`
+
+```yaml
+accounts:
+  - name: alice
+    coins: ["1000token", "1000000000stake"]
+  - name: bob
+    coins: ["500token"]
+validator:
+  name: alice
+  staked: "100000000stake"
+faucet:
+  host: "0.0.0.0:4501"
+  name: bob
+  coins:
+    - "5token"
+host:
+  rpc: "0.0.0.0:26659"
+  p2p: "0.0.0.0:26658"
+  prof: "0.0.0.0:6061"
+  grpc: "0.0.0.0:9091"
+  api: "0.0.0.0:1318"
+  frontend: "0.0.0.0:8081"
+  dev-ui: "0.0.0.0:12346"
+genesis:
+  chain_id: "mars"
+init:
+  home: "$HOME/.mars"
+```
+
+Use a  different terminal window for each blockchain. Run the following commands for each blockchain:
+
+```
+starport serve -c earth.yml
+starport serve -c mars.yml
+```
+
+### Start the Relayer
+
+First, configure the relayer. Use the Starport `configure` command with the `--advanced` option:
```

Comment:
```suggestion
First, configure the relayer. Use the Starport `configure` command with the `--advanced` option:
```

Reply:
What's the suggested change here? I cannot spot the difference

Reply:
Might've been a mistake, please, ignore 🙂